docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -0,0 +1,318 @@
+
+== Example: Target pods by host and label
+
+To automatically identify a Redis Pod and monitor it with the Redis integration, uncomment the following input configuration inside  the {agent} Standalone manifest [here](https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml):
+
+
+[source,yaml]
+------------------------------------------------
+- name: redis
+  type: redis/metrics
+  use_output: default
+  meta:
+    package:
+      name: redis
+      version: 0.3.6
+  data_stream:
+    namespace: default
+  streams:
+    - data_stream:
+        dataset: redis.info
+        type: metrics
+      metricsets:
+        - info
+      hosts:
+        - '${kubernetes.pod.ip}:6379'
+      idle_timeout: 20s
+      maxconn: 10
+      network: tcp
+      period: 10s
+      condition: ${kubernetes.labels.app} == 'redis'
+------------------------------------------------
+
+The condition `${kubernetes.labels.app} == 'redis'` will make the {agent} look for a Pod with the `redis` label app within the scope defined in its manifest. 
+
+List of exported fields of the Kubernetes integration {metricbeat-ref}/exported-fields-kubernetes-processor.html[here] to be used in conditions.
+
+The `redis` input defined in the {agent} manifest only has `info` as metricset. You can find more metricsets in the {metricbeat-ref}/metricbeat-module-redis.html[Redis module page], as well as the configuration options.
+
+To deploy Redis, you can apply the following example manifest:
+
+[source,yaml]
+------------------------------------------------
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-redis-config
+data:
+  redis-config: |
+    maxmemory 2mb
+    maxmemory-policy allkeys-lru
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: redis
+  labels:
+    k8s-app: redis
+    app: redis
+spec:
+  containers:
+  - name: redis
+    image: redis:5.0.4
+    command:
+      - redis-server
+      - "/redis-master/redis.conf"
+    env:
+    - name: MASTER
+      value: "true"
+    ports:
+    - containerPort: 6379
+    resources:
+      limits:
+        cpu: "0.1"
+    volumeMounts:
+    - mountPath: /redis-master-data
+      name: data
+    - mountPath: /redis-master
+      name: config
+  volumes:
+    - name: data
+      emptyDir: {}
+    - name: config
+      configMap:
+        name: test-redis-config
+        items:
+        - key: redis-config
+          path: redis.conf
+------------------------------------------------
+
+You should now be able to see Redis data flowing in on index `metrics-redis.info-default`. Make sure that the port on your Redis manifest file and the one used in the Redis input match. 
+
+**Note:** All assets (dashboards, ingest pipelines, and so on) related to the Redis integration are not installed. You need to explicitly <<install-uninstall-integration-assets,install them through Kibana>>.
+
+== Example: Target pods by label
```

Comment:
@ChrsMark and @gizas do you understand what is the difference between `Target pods by label` and `Target pods by host and label` example?

Reply:
`Target pods by host and label` example looks the same with `Target pods by label` example. The only difference is that the Redis example uses `${kubernetes.pod.ip}` as a host but this should not identify the example since it could be any value coming from the metadata. I would avoid making so explicit here.